src/labels/CustomLabels.labels

```diff
@@ -54,7 +54,7 @@
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>CannotDelete</shortDescription>
-        <value>Record cannot be deleted because children records exist. Delete children first.</value>
+        <value>This record can't be deleted because it has one or more related records associated with it. Delete the associated records first, then delete this record. To determine which records should be deleted first, click the Related tab. If you don't see related records, or if you're unable to delete them, contact your System Administrator for assistance.</value>
```

Comment:
@xinnanlisf instead of using the literal single quote and we change it to the escaped version &apos; ? Son from "don't" to "don&apos;t"

Reply:
Good catch! Should be good now.